File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/host/exception/NotPartnerHostException.java
package band.gosrock.domain.domains.host.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class NotPartnerHostException extends DuDoongCodeException {
    public static final DuDoongCodeException EXCEPTION = new NotPartnerHostException();

    private NotPartnerHostException() {
        super(HostErrorCode.NOT_PARTNER_HOST);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/host/exception/NotPartnerHostException.kt
package band.gosrock.domain.domains.host.exception

import band.gosrock.common.exception.DuDoongCodeException

class NotPartnerHostException private constructor() : DuDoongCodeException(HostErrorCode.NOT_PARTNER_HOST) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = NotPartnerHostException()
    }
}
